k8s/scripts/ci_minikube_setup.sh

```diff
@@ -17,22 +17,41 @@ OS=$( uname -s | tr '[:upper:]' '[:lower:]' )
 
 MINIKUBE_IP=$( minikube ip )
 
-# Generate coredns configMap patch
-echo '        hosts {' > $PATCH
-for service in $SERVICES; do
-    echo "          $MINIKUBE_IP $service.$DOMAIN" >> $PATCH
-done
-echo '          fallthrough' >> $PATCH
-echo '        }' >> $PATCH
-
 # Patch coredns configMap
+echo 'Patching coredns configMap (adding custom service entries to the hosts section)...'
 if ! kubectl get cm coredns -n kube-system -o yaml | grep -q hosts; then
-    echo 'Patching coredns ConfigMap'
+    echo 'hosts section does not exist, adding it'
+
+    # Generate coredns configMap patch
+    echo '        hosts {' > $PATCH
+    for service in $SERVICES; do
```

Comment:
move host-list generation to a function, as it is repeated twice.

Reply:
done